test/unit/carriers/canada_post_test.rb

```diff
@@ -130,9 +132,4 @@ def test_delivery_range_with_invalid_date
 
     assert_equal [], rate_estimates.rates[0].delivery_range
   end
-
-  def test_line_items_with_nil_values
-    @line_items << Package.new(500, [2, 3, 4], :description => "another box full of stuff", :value => nil)
-    @carrier.find_rates(@origin, @destination, @line_items)
-  end
```

Comment:
I'm assuming this test used to check if an exception was thrown. I'd rather we keep stub it out the same way we do other similar tests and assert no exception is raised.